frontend/src/routes/_account.plan.index.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright 2025 New Vector Ltd.
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+// Please see LICENSE in the repository root for full details.
+
+import { queryOptions, useSuspenseQuery } from "@tanstack/react-query";
+import { Navigate, createFileRoute } from "@tanstack/react-router";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { graphql, useFragment } from "../gql";
+import { graphqlRequest } from "../graphql";
+
+export const CONFIG_FRAGMENT = graphql(/* GraphQL */ `
+  fragment PlanManagement_siteConfig on SiteConfig {
+    planManagementIframeUri
+  }
+`);
+
+const QUERY = graphql(/* GraphQL */ `
+  query SiteConfig {
+    siteConfig {
+      ...PlanManagement_siteConfig
+    }
+  }
+`);
```

Comment:
You don't need to have a fragment here, fragments are for "leaf" components

Also please name this query with a unique name, and unique query key that reflects that this is for the plan management page